library/initial-content/download_initial_content.gradle

```diff
@@ -7,9 +7,9 @@ android.libraryVariants.all { variant ->
     def mobileContentApi = variant.flavorName.contains("stage") ? uris.mobileContentApi.stage : uris.mobileContentApi.production
 
     // include the list of downloaded tools and languages as a BuildConfig constant
-    variant.buildConfigField('java.util.Set<String>', 'BUNDLED_TOOLS', 'com.google.common.collect.ImmutableSet.of('
+    variant.buildConfigField('java.util.List<String>', 'BUNDLED_TOOLS', 'java.util.Arrays.asList('
             + bundledContent.tools.collect{ "\"$it\"" }.join(',') + ')')
-    variant.buildConfigField('java.util.Set<String>', 'BUNDLED_LANGUAGES', 'com.google.common.collect.ImmutableSet.of('
+    variant.buildConfigField('java.util.List<String>', 'BUNDLED_LANGUAGES', 'java.util.Arrays.asList('
```

Comment:
`ImmutableSet` is in the guava library, it was the only thing we were using from the library for initial content, and I've been actively trying to remove it. So I opted for the next best option of using `Arrays.asList` from the core java SDK

Reply:
`ImmutableSet` was originally used because the contents of the set/list shouldn't be modified, but enforcing that isn't worth the overhead of including the guava library

Reply:
Makes sense, cleaning up and removing stuff is always good.